core/tests/protractor_utils/forms.js

```diff
@@ -601,17 +627,17 @@ var CodeMirrorChecker = function(elem, codeMirrorPaneToScroll) {
    * This recursive function is used by expectTextWithHighlightingToBe().
    * currentLineNumber is the current largest line number processed,
    * scrollTo is the number of pixels from the top of the text that
-   * codemirror should scroll to,
+   * codeMirror should scroll to,
    * codeMirrorPaneToScroll specifies the CodeMirror's left or right pane
    * which is to be scrolled.
    * compareDict is an object whose keys are line numbers and whose values are
    * objects corresponding to that line with the following key-value pairs:
    *  - 'text': the exact string of text expected on that line
    *  - 'highlighted': true or false, whether the line is highlighted
    *  - 'checked': true or false, whether the line has been checked
-   * compareHightlighting: Whether highlighting should be compared.
+   * compareHighLighting: Whether highlighting should be compared.
    */
-  var _compareText = async function(compareDict, compareHightlighting) {
+  var _compareText = async function(compareDict, compareHighLighting) {
```

Comment:
Did you actually change the name of this variable (I don't think I've seen it in the list of changes)?

Reply:
only changed the name of variable for spelling error.